inspirehep/modules/records/testsuite/test_records.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of INSPIRE.
-# Copyright (C) 2015 CERN.
+# Copyright (C) 2015, 2016 CERN.
```

Comment:
Shall we move this test to the top level `tests/` folder? You can create a subfolder `records` there if we want to better separate tests by functionality.